spec/fixtures/stacks/rev1_07/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = "~> 1.7.0"
+  required_providers {
+    null = {
+      version = "~> 3.0"
```

Comment:
Minor: This is just a comment for next time. AWS provider 3 is a couple versions old now so we can start testing with newer providers when we add future Terraform versions.